LiberateAutomation/src/com/liberate/automation/pom/MaintainDepositReason.java

```diff
@@ -47,4 +52,109 @@
+		this.action = action;
+	}
+	
+	/***
+	 * The method to navigate to Maintain Deposit Reason screen
+	 * 
+	 * @return Returns true if able to navigate, else will return false.
+	 */
+	public boolean navigate() {
+		boolean passed = false;
+
+		passed = action.waitFor(LiberateCommon.LevelOne.Orders, 4, true);
+		passed = action.clickOn(LiberateCommon.LevelOne.CreditControl);
+		passed = action.waitFor(LiberateCommon.CreditControl.DepositMaintenance, 4, true);
+		passed = action.clickOn(LiberateCommon.CreditControl.DepositMaintenance);
+		passed = action.waitFor(LeftLink.DepositMaintenance.MaintainDepositReason, 4, true);
+		passed = action.clickOn(LeftLink.DepositMaintenance.MaintainDepositReason);
+
+		return passed;
+	}
+
+	
+	
+	/***
+	 * Method to Click on'New' command button to navigate to Maintain Deposit Reason - New Functionality
+	 * 
+	 * @return True, if able to select; false if not able to select.
+	 */
+	public boolean clickNewButton() {
+		boolean passed = false;
+
+		passed = action.waitFor(NewButton, 4, true);
+		passed = action.clickOn(NewButton);
+
+		return passed;
+	}
+	
+	
+	/***
+	 * Method to create New Maintain Deposit Reason Functionality
+	 * 
+	 * @return True, if able to select; false if not able to select.
+	 */
+	public boolean maintainDepositReason_CreateNew(String DepositReasonCode, String Description, String InterestToBeCreditedToCustomersBill,
+	String NoofMonthsOverWhichDepositIsRefunded) {
+		boolean passed = false;
+		
+		passed = action.waitFor(Acceptbutton, 4, true);
+		passed = action.sendDataTo(DepositReasonCode_inputField, DepositReasonCode);
+		passed = action.sendDataTo(Description_inputField, Description);
+		passed = action.clickOn(RefundDeposit_radioButtonYes);
+		passed = action.clickOn(AccrueInterestOnDeposit_radioButtonYes);
+		passed = action.clickOn(AutomaticallyRefundtoWorkingAccounts_Yes);
+		passed = action.sendDataTo(InteresttobeCreditedtoCustomersBill, InterestToBeCreditedToCustomersBill);
+		passed = action.clickOn(InterestisCompound_Yes);
+		passed = action.sendDataTo(NoofMonthsOverWhichDepositisRefunded, NoofMonthsOverWhichDepositIsRefunded);
+		passed = action.clickOn(PayInitialPartPeriodInterest_No);
+		passed = action.clickOn(UsageAdvance_No);
+		passed = action.clickOn(AutomaticallyRefundtoCeasedAccounts_No);
+		passed = action.clickOn(RefundMethod_Adjustment);
+				
+		return passed;
+	}
+	
+	
+	/***
+	 * Method to Click on 'Accept' button in Maintain Deposit Reason - New Functionality
+	 * 
+	 * @return True, if able to select; false if not able to select.
+	 */
+	public boolean acceptButton() {
+		boolean passed = false;
+		
+		passed = action.waitFor(Acceptbutton, 4, true);
+		passed = action.clickOn(Acceptbutton);
+		
+		return passed;
+	}
+	
+	
+	/***
+	 * Method to Click on 'Cancel' button in Maintain Deposit Reason - New Functionality
+	 * 
+	 * @return True, if able to select; false if not able to select.
+	 */
+	public boolean cancelButton() {

```

Comment:
Same Comment is applicable for all methods, regarding method name.
method name should be an action. 
eg: 'clickOnCancelButton'